.github/workflows/run-e2e.yaml

```diff
@@ -79,34 +63,35 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: eu-west-1
 
-      - id: install-yq-jq
-        if: github.event.inputs.environment == 'staging'
-        name: Install yq and jq to parse config file
+      - id: install
+        name: Install dependencies
         run: |-
-          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CC86BB64
-          sudo add-apt-repository ppa:rmescandon/yq
-          sudo apt update
-          sudo apt install yq -y
-          sudo apt-get install jq -y
+          cd e2e/
+          git config --global url."https://".insteadOf ssh://
+          npm ci
 
-      - id: copy-experiments
-        if: github.event.inputs.environment == 'staging'
-        name: Copy experiments
+      - id: set-up-creds
+        name: Set up Kubernetes credentials.
         run: |-
-          cd biomage-utils/
-          for EXP_ID in $(yq e '.experiment-ids' ../config.yaml -j | jq '.[]')
-          do
-            biomage experiment copy -e $(echo $EXP_ID | sed s'/"//g')
-          done
+          aws eks update-kubeconfig --name biomage-$ENV
```

Comment:
We need to make sure this is saved in `iac` somewhere. Currently the rights of the CI runner are manually assigned by me.